portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,7 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function changeBackgroundColor(color) {
```

Comment:
forgot to mention, per the [styleguide](https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes), could you add a JS doc on top of the function please? (description of what the method does, what parameter and type it receives, etc..)?